src/main/java/org/sonarsource/sonarlint/ls/CommandManager.java

```diff
@@ -164,12 +175,26 @@ public List<Either<Command, CodeAction>> computeCodeActions(CodeActionParams par
     return codeActions;
   }
 
+  private void createFixWithAiCodeFixCodeAction(UUID issueId, List<Either<Command, CodeAction>> codeActions, Diagnostic diagnostic, URI fileUri, String message) {
```

Comment:
Note for later: I'm starting to think that maybe we should refactor the creation and handling of the code actions out of the `CommandManager` (in separate classes maybe?), the class is starting to be quite large.

Reply:
Agreed. I have created https://sonarsource.atlassian.net/browse/SLLS-322